src/move/move.spec.ts

```diff
@@ -0,0 +1,161 @@
+import exp from "constants";
+import { Board } from "../board/board";
+import { initGameState } from "../game/state";
+import { initRandomGame } from "../randomizer/randomizer";
+import { checkUserMove } from "./move";
+import { GameState } from "../game/state";
+import { Tile } from "../tile/tile";
+import { BLACK_PLAYER, WHITE_PLAYER } from "../player/player";
+
+const mockBoard: Board = [
+  [
+    undefined,
+    undefined,
+    undefined,
+    { symbol: "E", color: "blue" },
+    { symbol: "A", color: "yellow" },
+    { symbol: "A", color: "blue" },
+    { symbol: "O", color: "blue" },
+    undefined,
+    undefined,
+    undefined,
+  ],
+  [
+    undefined,
+    undefined,
+    { symbol: "A", color: "magenta" },
+    { symbol: "A", color: "cyan" },
+    { symbol: "A", color: "green" },
+    { symbol: "D", color: "red" },
+    { symbol: "O", color: "green" },
```

Comment:
@paulSanchezMarmelab , a tile cannot has this value: O must be unique in the board with grey color

Reply:
Yes I saw that, and it broke mock board for tests. It should be fixed now.